inject/android.js

```diff
@@ -2,9 +2,23 @@
  * @module Android integration
  * @category Content Scope Scripts Integrations
  */
-import { load, init } from '../src/content-scope-features.js'
+import { load, init, update } from '../src/content-scope-features.js'
 import { processConfig, isGloballyDisabled } from './../src/utils'
 
+const allowedMessages = [
```

Comment:
Can we drop a utils/messaging file into https://github.com/duckduckgo/content-scope-scripts/tree/main/inject or similar and remove all this copy and paste?

Reply:
I copied this from inject `chrome.js`, @kzar would be best to comment what we do here

Reply:
We can leave for now and just file a clean up PR, a lot of this is on the extension side too (which is my fault https://app.asana.com/0/1201614831475344/1204299256427937/f) we could clean them all up as a single task perhaps.

Just so long as we're all aware that it's in 4 places is fine.